R/report_null.R

```diff
@@ -0,0 +1,72 @@
+#' Creates `NULL` Report
+#'
+#' @param tlg to convert to null report.
+#' @param ind (`integer`) indentation for the outputs of class `VTableTree`.
+#' @param ... not used.
+#'
+#' @rdname report_null
+#' @name report_null
+#'
+#' @returns the `tlg` object or a `NULL` report if the `tlg` is `NULL`, is a `TableTree` with 0 rows, is a `listing_df`
+#'   with 0 rows or is a `list` with 0 elements.
+#'
+#' @export
+#' @examples
+#' report_null(NULL)
+setGeneric("report_null", function(tlg, ...) {
```

Comment:
`...` are not casted down to functions so I think we can drop them

Reply:
I think we need it since some methods have an additional argument.

Reply:
If there are no uses for `...`, you should drop them. The extra parameter (the inset) should be dropped, too, if not used in the internal calls. You should not keep it if you want to follow the idea of a standard TLG, as it will never be used otherwise and people would always change it in post-processing with `table_inset`

Reply:
`report_null` is an important tool for the users. It is not an internal function, the `inset` has to be parametrizable. if one does not want to have a different set of parameters depending on the method (the method for `ANY` doesn't have the `ind` parameter), it is mandatory to have `...` in the parameter of the generic.